apps/daffio/src/app/content/pwa/pages/pwa/pwa.component.html

```diff
@@ -0,0 +1,159 @@
+<daff-hero color="accent" class="hero" size="small">
+  <daff-container size="xl">
+    <div class="hero__grid">
+      <div class="hero__text">
+        <h1 daffHeroTitle>
+          Build the <span>next generation</span> of ecommerce stores.
+        </h1>
+        <h2 daffHeroSubtitle>
+          Progressive Web Apps take advantage of the best features of web and native applications.
+        </h2>
+      </div>
+      <img class="hero__image" src="/assets/why-pwa/pwa-illustration.svg" />
+    </div>
+  </daff-container>
+</daff-hero>
+
+<daff-callout layout="centered" class="features">
+  <daff-container size="md">
+    <h3 daffCalloutTitle>
+      A new level of quality to help scale your business
+    </h3>
+    <p daffCalloutSubtitle>
+      Progressive Web Apps are a new way to offer incredible mobile shopping experiences that are fast, reliable and accessible through the web.
```

Comment:
How do you feel about the Oxford Comma? I'd prefer having it --> "fast, reliable, and accessible"